ngraph/frontend/onnx_import/src/op/org.openvinotoolkit/group_norm.cpp

```diff
@@ -41,13 +42,17 @@ namespace ngraph
                     {
                         const auto& pshape = data.get_partial_shape();
                         size_t rank_size = pshape.rank().get_length();
-                        NGRAPH_CHECK(rank_size == 4, "Only 4-D input tensors supported");
+                        NGRAPH_CHECK(rank_size >= 4, "4-D and above tensors supported only");
 
                         if (pshape.is_static())
                         {
                             const auto& shape = pshape.to_shape();
                             std::vector<size_t> new_shape{
                                 shape[0], num_groups, shape[1] / num_groups, shape[2], shape[3]};
+                            for (size_t i = 4; i < rank_size; i++)
+                            {
+                                new_shape.push_back(shape[i]);
+                            }
```

Comment:
```suggestion
                        NGRAPH_CHECK(rank_size >= 2, "2-D and above tensors supported only");

                        if (pshape.is_static())
                        {
                            const auto& shape = pshape.to_shape();
                            std::vector<size_t> new_shape{
                                shape[0], num_groups, shape[1] / num_groups};
                            for (size_t i = 2; i < rank_size; i++)
                            {
                                new_shape.push_back(shape[i]);
                            }
```

Reply:
why do we need 2-d or 3-d tensors in group norm?

Reply:
If it's not prohibited by the spec(if there is any) it should be allowed.
3D tensors can be useful for text data or sound wave for example.
2D tensors are not useful at all in this context and can be prohibited in the line no. 55.
There is no need to write indexes 2 and 3 by hand if this can be handled by this for loop if you are afraid of performance of 2 more push_backs then you can reserve the memory for it.

Reply:
> 2D tensors are not useful at all in this context and can be prohibited in the line no. 55.
so the line should be rather look like:
NGRAPH_CHECK(rank_size >= 3, "3-D and above tensors supported only");